keras_retinanet/bin/evaluate.py

```diff
@@ -2,13 +2,10 @@
 
 """
 Copyright 2017-2018 Fizyr (https://fizyr.com)
-
```

Comment:
Why are these removed?

Reply:
I have no idea why did it happened, will check